src/report-filter-controls.tsx

```diff
@@ -170,12 +141,10 @@ function ReportFilterControls({
         </div>
         {env !== DEFAULT_ENV && <input type="hidden" name="env" value={env} />}
       </form>
-      <ReportFilterControlsContext.Provider value={filterControls}>
-        {children}
-      </ReportFilterControlsContext.Provider>
+      {children}
```

Comment:
We don't need `children` anymore, do we?

Reply:
I guess not! I thought it was convention/habit to always allow children in a component